resources/images/bitcoin/insecure/Dockerfile_v0.16.1

```diff
@@ -0,0 +1,113 @@
+# Setup deps stage
+FROM alpine:3.7 AS deps
+ENV REPO="bitcoin/bitcoin"
+ENV COMMIT_SHA="dac5d68fc6cf136e0d7b21b9ed4fa053d54e6059"
+ENV BUILD_ARGS="--disable-tests --without-gui --disable-bench --disable-fuzz-binary --enable-suppress-external-warnings"
```

Comment:
unclear to me where we are tracking our "defaults." seems fragile to be repeating them all in each image, and also now there is a discrepancy between these images and the fixes I did for https://github.com/bitcoin-dev-project/warnet/issues/441

if you have any ideas on how to make "one source of truth" for our docker configs that can then be overridden by the user, it would be nice to handle that, either here or in a separate PR.

however, at the very least I think you should include the changes for disabling dependency tracking and making gcc less resource hungry in this PR.

Reply:
We aren't tracking defaults as far as I know. Different builds need different things to get working in roughly the same way